lib/bibliothecary/parsers/maven.rb

```diff
@@ -24,6 +30,10 @@ def self.mapping
             kind: 'lockfile',
             parser: :parse_ivy_report
           },
+          /^tidelift-gradle-resolved\.txt|.*\/tidelift-gradle-resolved\.txt$/i => {
+            kind: 'manifest',
```

Comment:
if this includes transitive deps already (I think it does) then it should be lockfile instead so we know not to try to resolve transitive ourselves

Reply:
I had it as a lockfile, and then I saw "com.fasterxml.jackson.core:jackson-annotations:2.9.0 -> 2.9.6". Which I thought indicated that it was more of a manifest.

Reply:
the `-> 2.9.6` is the resolution AIUI (2.9.6 is conceptually the lockfile version / 2.9.0 the manifest version)